062-support-tiered-storage.md

```diff
@@ -0,0 +1,121 @@
+#  Support tiered storage natively in Strimzi
+
+## Current situation
+
+Kafka introduced tiered storage feature since 3.6. The feature is in early stage and it’s nice to support tiered storage configuration natively via Strimzi.
+
+## Motivation
+
+Supporting tiered storage via Strimzi allow Strimzi users to enable tiered storage natively with Strimzi config. By allowing to config custom configuration for RemoteStorageManager and RemoteLogMetadataManager, users are able to adopt tiered storage feature on Strimzi easily.
+
+It should also be noted that, it’s on the end-user to ensure that their tiered storage setup works with Strimzi; the operator itself is strictly responsible for pushing down the necessary config to the broker pods.
```

Comment:
Might be worth explaining that this proposal focuses on tiered storage support with a custom plugin brought in by the user. But other proposals in the future might add some built-in support for selected storage implementations such as S3. That will add some more clarity for users reading it.

Reply:
Done